### PR TITLE
Update `requirements.txt` for `gradio` and `gradio_client`

### PR DESCRIPTION
**Pull request by abidlabs**
Closes: https://github.com/gradio-app/gradio/issues/10085
Closes: https://github.com/gradio-app/gradio/issues/10084

### CODE REVIEW

File: client/python/requirements.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ httpx>=0.24.1
 huggingface_hub>=0.19.3
 packaging
 typing_extensions~=4.0
-websockets>=10.0,<13.0
+websockets>=10.0,<15.0
```
Comment (abidlabs) [inline client/python/requirements.txt:6]: I'd like to do a 2.0 `gradio_client` release soon where we drop `websockets` as a dependency and end support for 3.x altogether. But in the meantime

File: requirements.txt [hunk 2]
```diff
@@ -18,7 +18,7 @@ python-multipart==0.0.12 # required for fastapi forms. pinning to avoid yanking
 pydub
 pyyaml>=5.0,<7.0
 ruff>=0.2.2; sys.platform != 'emscripten'
-safehttpx>=0.1.1,<1.0
+safehttpx>=0.1.5,<0.2.0
```
Comment (abidlabs) [inline requirements.txt:21]: Mainly to get this out: https://github.com/gradio-app/safehttpx/pull/4

**Review by aliabid94 — APPROVED** [review]
Tested with updated libraries, LGTM

**Comment by abidlabs** [comment]
Thanks @aliabid94!

**Pull request merged**